### PR TITLE
Redirect for kubernetes docs

### PR DESCRIPTION
**Pull request by nottrobin**
For @evilnick

QA
--

`./run` and go to http://0.0.0.0:8001/kubernetes/docs - see you're redirected to the beautiful new Kubernetes docs at http://blog.juju.solutions/kubernetes-docs/.

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -683,3 +683,5 @@ usn/?: "https://usn.ubuntu.com/"
 (?P<page>.*)\.html: "/{page}"
 telco/?: "/telecommunications"
 telcos/?: "/telecommunications"
+kubernetes/docs/?: "http://blog.juju.solutions/kubernetes-docs/"
```
Comment (anthonydillon) [inline redirects.yaml]: Shouldn't we link to https?

Comment (evilnick) [reply redirects.yaml]: good point, I think that would be preferable

Comment (nottrobin) [reply redirects.yaml]: Oh I didn't realise that works

Comment (nottrobin) [reply redirects.yaml]: Fixed

**Pull request merged**